cdap-docs/admin-manual/source/index.rst

```diff
@@ -115,9 +115,9 @@ installation and its security configuration.
 
     - |logging|_ CDAP collects **logs** for all of its internal services and user
       applications; at the same time, CDAP can be **monitored through external systems**.
-      Covers **log location**, **logging messages**, the **master services logback
-      configuration** and **CDAP support for logging** through standard SLF4J (Simple
-      Logging Facade for Java) APIs.
+      Covers **log location**, **logging messages**, the **system services and user
+      application logback configuration** and **CDAP support for logging** through standard
+      SLF4J (Simple Logging Facade for Java) APIs and Logback.
```

Comment:
Yes; were you thinking internal links, such as such as to ``log location``, ``logging messages``, ``system services and user
application logback configuration``, and ``CDAP support for logging``?